### DISCUSSION TITLE
How to Manage Licenses

### DISCUSSION THREAD

**Discussion by KMNGC** [snipe/snipe-it#17557] [Q&A]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I could not find a good example in the documentation but basically, what is the concept of adding licenses in Snipe-IT?  
Lets say I create a license for something like Adobe Photoshop.  That license is good for one year, and then I need to renew it next year.  What is the process when it comes time to renew it?  Do I just create a new license, and then delete the old one?  Is there anyway to keep the existing license, and just replace the important information such as the product key, purchase date, etc. and then be able to go back and see the history of the values that were in some of those fields?  Just curious how others are doing it.

### Reproduction steps

1.  Create a new license.
2. Attempt to update that license with new information when it is time to be renewed / replaced (product key, seats, etc)
3.Attempt to view the history of certain fields after changing the information after you have renewed such as the Product Key from before, the order number, the expiration date, etc.  All you can see are the most up to date values, but you cannot see previous values in those fields.  

### Expected behavior

I would expect to see the history of that license from start to finish, instead of having to create a new license every single year or whenever.

### Snipe-IT Version

6.2.1

### Operating System

RHEL 8

### Web Server

Apache

### PHP Version

7.4.33

### Operating System

RHEL 8

### Browser

Firefox

### Version

N/A

### Device

N/A

### Operating System

N/A

### Browser

N/A

### Version

N/A

### Error messages

```shell
N/A
```

### Additional context

Neither fresh install or upgrade, just the version I've been messing around with.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.

**Comment by mauro-miatello** [comment]
I'm interested too. But it's not clear to me why you propose to create a new license instead of update expiry date.
